docs/architecture/cloud-native/index.md

```diff
@@ -41,33 +41,35 @@ Author:
 >
 > **Rob Vettor** - Microsoft - Principal Cloud System Architect/IP Architect - [RobVettor.com](http://robvettor.com)
 
-Editors:
-
-> **John Doe**
-
-## Introduction
+Participants and Reviewers:
 
-TODO
+> **Cesar de la Torre**, Sr. PM, .NET product team, Microsoft Corp.
+>
+> **Nish Anil**, Sr. Program Manager, .NET team, Microsoft
 
-## Purpose
+Editors:
 
-TODO
+> **Maira Wenzel**, Content Developer, .NET team, Microsoft
```

Comment:
if you're adding seniority, you might as well add mine 😉 thanks for including me!

```suggestion
> **Maira Wenzel**, Sr. Content Developer, .NET team, Microsoft
```